static/oauth/oauthHandler.js

```
@@ -0,0 +1,34 @@
+const bpmnServerURL = 'https://wfsdev.p10.io'
```

Do not hard-code here, move this inside nuxt.config.js. Read from ENV first with fallback to the hard-coded value.